### PR TITLE
add Pageable, swager

### PR DESCRIPTION
**Pull request by YuraNR**

### CODE REVIEW

File: src/main/java/bookstore/controller/BookController.java [hunk 1]
```diff
@@ -26,30 +31,40 @@ public class BookController {
 
     @ResponseStatus(HttpStatus.OK)
     @GetMapping
-    public List<BookDto> getAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books",
+            description = "Get a list of all books")
+    public List<BookDto> getAll(@PageableDefault(page = 0, size = 2) Pageable pageable) {
```
Comment (SalnikovMarkiian) [inline src/main/java/bookstore/controller/BookController.java:36]: u can also add direction of sorting into parameters by default:
`direction = Sort.Direction.ASC`

File: src/main/java/bookstore/controller/BookController.java [hunk 2]
```diff
@@ -26,30 +31,40 @@ public class BookController {
 
     @ResponseStatus(HttpStatus.OK)
```
Comment (SalnikovMarkiian) [inline src/main/java/bookstore/controller/BookController.java:32]: This status is returned by default so you may remove it in all endpoints

**Review by SalnikovMarkiian — APPROVED** [review]
GREAT! I approved ur solution, but pay attention to my comments

**Review by NazarSmith228 — APPROVED** [review]
LGTM

**Pull request merged**